Tasks/MavenV3/maventask.ts

```diff
@@ -155,65 +159,64 @@ async function execBuild() {
             if (authenticateFeed) {
                 var repositories;
 
-                if (skipEffectivePomGeneration)
-                {
+                if (skipEffectivePomGeneration) {
                     var pomContents = fs.readFileSync(mavenPOMFile, "utf8");
                     repositories = util.collectFeedRepositories(pomContents);
                 } else {
                     var mvnRun = tl.tool(mvnExec);
                     mvnRun.arg('-f');
                     mvnRun.arg(mavenPOMFile);
                     mvnRun.arg('help:effective-pom');
-                    if(mavenOptions) {
+                    if (mavenOptions) {
                         mvnRun.line(mavenOptions);
                     }
-                    repositories = util.collectFeedRepositoriesFromEffectivePom( mvnRun.execSync()['stdout']);
+                    repositories = util.collectFeedRepositoriesFromEffectivePom(mvnRun.execSync()['stdout']);
                 }
                 return repositories
-                .then(function (repositories) {
```

Comment:
Just to double-check - there were some indent changes only?

Reply:
Yes, only the indents